libraries/botbuilder-dialogs/src/prompts/confirmPrompt.ts

```diff
@@ -21,33 +24,11 @@ import { Culture } from '@microsoft/recognizers-text';
 export class ConfirmPrompt extends Prompt<boolean> {
 
     /**
-     * Default confirm choices for a range of locales.
-     * @deprecated since version 4.3
-     */
-    private static defaultConfirmChoices: { [locale: string]: (string | Choice)[] } = {
```

Comment:
again this is technically a breaking change.

Reply:
resolved per email. property is private